client/src/components/FormsUI/Button/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import { useFormikContext } from 'formik';
+
+interface Props {
+  color: 'primary' | 'secondary' | 'inherit' | 'default';
+  children?: React.ReactNode;
+  onClick?: (values: any) => void;
+  type: string;
+  size: 'large' | 'medium' | 'small';
+  variant: 'text' | 'contained' | 'outlined';
+  className?: string;
+  fullWidth?: boolean;
+}
+
+export const ButtonWrapper: React.FC<Props> = ({ color, size, variant, fullWidth, children }) => {
```

Comment:
We probably didn't need to componentize this, as we are returning only a Button :)